controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1794,6 +1799,25 @@ bool master_thread::handle_cmdu_1905_topology_response(const std::string &src_ma
     return true;
 }
 
+bool master_thread::handle_cmdu_1905_backhaul_sta_steering_response(
+    const std::string &src_mac, ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    auto mid = cmdu_rx.getMessageId();
+    LOG(DEBUG) << "Received BACKHAUL_STA_STEERING_MESSAGE from " << src_mac << ", mid=" << std::hex
+               << mid;
+
+    auto tlv_backhaul_sta_steering_resp = cmdu_rx.getClass<wfa_map::tlvBackhaulSteeringResponse>();
+    if (!tlv_backhaul_sta_steering_resp) {
+        LOG(ERROR) << "Failed getClass<wfa_map::tlvBackhaulSteeringResponse>";
+        return false;
+    }
+
+    LOG(DEBUG) << "BACKHAUL_STA_STEERING_MESSAGE result_code: "
+               << int(tlv_backhaul_sta_steering_resp->result_code());
+
```

Comment:
The controller should respond with an ACK.

Reply:
Added sending ACK message to the agent and fixed commit message.